lib/util/string_ext.dart

```diff
@@ -74,4 +76,8 @@ extension StringExtension on String {
         return null;
     }
   }
+
+  bool get isPostcardId {
+    return split('-')[1] == Environment.postcardContractAddress;
```

Comment:
I'm not sure there will be some changes for the id format in the future. So can you please check `split('-')` size > 1 first?